server/src/test/java/org/elasticsearch/action/admin/indices/rollover/RolloverRequestTests.java

```diff
@@ -186,6 +189,48 @@ public void testValidation() {
         assertEquals("rollover target is missing", validationException.validationErrors().get(0));
     }
 
+    public void testParsingWithType() throws Exception {
+        final XContentBuilder builder = XContentFactory.jsonBuilder()
+            .startObject()
+                .startObject("conditions")
+                    .field("max_age", "10d")
+                    .field("max_docs", 100)
+                .endObject()
+                .startObject("mappings")
+                    .startObject("type1") //TO ASK did we allow types or only _doc?
```

Comment:
as per comment (to be deleted) the test previously allowed for type name to be present in mappings. But will we now allow for this as well or should we only allow `_doc`?